controls/combobox/styling.md

```diff
@@ -10,34 +10,36 @@ slug: combobox-styling
 
 The [Telerik UI for .NET MAUI ComboBox]({%slug combobox-overview%}) provides the following Style properties for customizing its look:
 
-* `PlaceholderColor` (`Microsoft.Maui.Graphics.Color`)&mdash;Defines the color for the placeholder text.
-* `TextColor` (`Microsoft.Maui.Graphics.Color`)&mdash;Defines the color of the text when the control is editable and the color of the selected item when the control is not editable and the selection mode is single.
-* `BackgroundColor` (`Microsoft.Maui.Graphics.Color`)&mdash;Defines the background color of the control.
-* `BorderColor` (`Microsoft.Maui.Graphics.Color`)&mdash;Defines the color of the border.
-* `BorderThickness` (`Microsoft.Maui.Thickness`)&mdash;Defines the thickness of the border.
-* `ClearButtonStyle` (of type `Style` with target type `Telerik.Maui.Controls.RadButton`)&mdash;Defines the style for the clear button.
-* `Font Options` (`FontAttributes`, `FontFamily`, `FontSize`)&mdash;Define the font options to the text of the RadComboBox. It's applied to the Placeholder, Selected Text (for single selection), and when the control is in Editable Mode.
-* `HighlightTextColor` (`Color`)&mdash;Specifies the color of the text that will be highlighted when searching (`IsEditable` must be `True`).
+* `PlaceholderColor`(`Color`)&mdash;Defines the color for the placeholder text.
+* `TextColor`(`Color`)&mdash;Defines the color of the text when the control is editable and the color of the selected item when the control is not editable and the selection mode is single.
```

Comment:
Can you split and simplify this sentence? For example:

* `TextColor`(`Color`)&mdash;Defines the following colors:
  * The color of the text when the control is editable.
  * The color of the selected item when the control is not editable and the selection mode is set to single.